js/index.js

```diff
@@ -1,42 +1,86 @@
+    'features-h4': 'Features',
+    'features-content': 'Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'about-h4': 'About',
+    'about-content': 'About content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'middle-img-src': 'img/mid-page-accent.jpg',
+    'services-h4': 'Services',
+    'services-content': 'Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'product-h4': 'Product',
+    'product-content': 'Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'vision-h4': 'Vision',
+    'vision-content': 'Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
   },
-  "contact": {
-    "contact-h4" : "Contact",
-    "address" : "123 Way 456 Street Somewhere, USA",
-    "phone" : "1 (888) 888-8888",
-    "email" : "sales@greatidea.io",
+  contact: {
+    'contact-h4': 'Contact',
+    address: '123 Way 456 Street Somewhere, USA',
+    phone: '1 (888) 888-8888',
+    email: 'sales@greatidea.io',
   },
-  "footer": {
-    "copyright" : "Copyright Great Idea! 2018"
+  footer: {
+    copyright: 'Copyright Great Idea! 2018',
   },
 };
 
 // Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+const logo = document.getElementById('logo-img');
+logo.setAttribute('src', siteContent.nav['img-src']);
+
+
+let newElementHolder = document.createElement('a');
+newElementHolder.href = '#';
+document.querySelector('nav').appendChild(newElementHolder);
+newElementHolder = document.createElement('a');
+newElementHolder.href = '#';
+document.querySelector('nav').prepend(newElementHolder);
+
+document.querySelectorAll('nav a').forEach((tag, index) => {
+  const element = tag;
+  element.textContent = siteContent.nav[`nav-item-${index}`];
+  element.style.color = 'green';
+});
+
+document.querySelector('.cta .cta-text h1').textContent = siteContent.cta.h1;
+document.querySelector('.cta .cta-text button').textContent = siteContent.cta.button;
+document.querySelector('#cta-img').src = siteContent.cta['img-src'];
+
+const topContent = document.querySelectorAll('.top-content .text-content');
+topContent[0].innerHTML = `<h4>${siteContent['main-content']['features-h4']}</h4>
```

Comment:
Just a heads up, while very useful innerHTML is super unsecure. Most of the time just using something like textContent will get the job done while keeping your codebase secure.